src/javascripts/components/eventListeners.js

```diff
@@ -1,13 +1,15 @@
 import $ from 'jquery';
 import chatBox from './chatBox/chatBox';
-
-window.jQuery = $;
-window.$ = $;
+import effects from '../helpers/effects';
 
 const eventListeners = () => {
   $('#chatInput').keyup(chatBox.newMessageEvent);
   $('#clearChat').click(chatBox.clearMessages);
+  $('#chatBox').on('click', '.editBtn', chatBox.editMessage);
+  $('#chatBox').on('click', '.saveBtn', chatBox.saveMessage);
   $('#chatBox').on('click', '.deleteBtn', chatBox.deleteMessage);
+  $('#chatBox').on('mouseenter', '.messageContainer', effects.messageMouseenter);
```

Comment:
I recommend changing the .messageContainer to .messageContent for the mousenter and mouseleave events. This change causes the animation to only occur if you hover your mouse over the actual message content instead of the messageContainer div. I think this looks better style wise but feel free to disagree.

Reply:
Actually, this change is problematic. it causes you to be unable to click on the edit and save buttons haha. If we can find a way to stop that from happening then that would be cool but its all just a style preference at the end of the day

Reply:
ok I think i figured it out. we add a class called .messageRow to the flex row div surrounding the messageContent and message buttons and call the mouseenter and mouseleaver on .messageRow. This seems to be the easiest solution I can come up with.

Reply:
```javascript
 domString += `<div id="${message.messageId}" class="messageContainer d-flex flex-column align-items-end text-right ml-2">`;
      domString += `<p class="messageDate">${message.timeStamp}</p>`;
      domString += '<div class="d-flex flex-row messageRow">';
      domString += `<p class="messageContent messageBubbleOut msg-cont-right">${message.messageContent}</p>`;
      domString += '<div id="messageBtns" class="d-flex flex-row">';
      domString += '<button class="saveBtn fas fa-save" style="display: none;" aria-label="Save Message"></button>';
      domString += '<button class="editBtn fas fa-pencil-alt" aria-label="Edit Message"></button>';
      domString += '<button class="deleteBtn fas fa-times" data-dismiss="alert" aria-label="Delete Message"></button>';
      domString += '</div>';
      domString += '</div>';
      domString += '</div>';

Reply:
1) I made that change and it works.  I tried to do this earlier and came across the same animation issue you first came across.  I'm happy you figured out a solution, because it annoyed me but I didn't want to spend more time on it so we could make our MVP goal.
2) i fixed that text bug you reported in Slack.

**Notes:** 
- **UI Change**: your suggestion only worked on the right side messages.  the left side messages kept triggering the event on the entire space of the div.  then i figured out the right side messages worked how we wanted because `align-items-end` was sizing `.messageContainer` based on the contents.  so i found that using `align-items-start` did the same thing for the left side messages.
- **Text bug**: the `getText` function fixes this problem.  i know most of the function looks like gobbledygook (to my untrained eyes, at least), but if I'm reading the documentation correctly, it essentially creates a `<p>` tag around our edited text; strips away every other HTML tag within the edited text; and returns clean, plain text `innerHTML`.  here's my source, for posterity: https://medium.com/@albertogasparin/getting-plain-text-from-user-input-on-a-contenteditable-element-b711aba2cb36

Reply:
Sweet. I’m gonna pull this down in an hour or two but this solution sounds great. Nice work man!

Reply:
Sweet. I’m gonna pull this down in an hour or two but this solution sounds great. Nice work man!